src/test/mir-opt/generator-tiny/rustc.main-{{closure}}.generator_resume.0.mir

```diff
@@ -37,16 +33,14 @@ fn main::{{closure}}#0(_1: std::pin::Pin<&mut [generator@$DIR/generator-tiny.rs:
 
     bb1: {
         _10 = move _2;                   // scope 0 at $DIR/generator-tiny.rs:19:16: 25:6
-        nop;                             // scope 0 at $DIR/generator-tiny.rs:20:13: 20:15
-        (((*(_1.0: &mut [generator@$DIR/generator-tiny.rs:19:16: 25:6 {u8, HasDrop, ()}])) as variant#3).0: HasDrop) = HasDrop; // scope 0 at $DIR/generator-tiny.rs:20:18: 20:25
```

Comment:
Looks like this `HasDrop` value got optimized out completely? I guess that's what's causing the above. I'm not sure how significant that is to this test.